controllers/metal3.io/baremetalhost_controller.go

```diff
@@ -815,6 +836,11 @@ func (r *BareMetalHostReconciler) actionPreparing(prov provisioner.Provisioner,
 		return result
 	}
 
+	// store settings updated after cleaning
+	if err = r.storeHostFirmwareSettings(prov, info); err != nil {
+		info.log.Info("could not store firmware settings for host after cleaning", "error", err)
```

Comment:
In case of error it is expect to take some action, like an `actionError` or `actionFailed` (and to trigger the retry mechanism for transient error, there is the `recordActionFailure` helper method)